scripts/generate-sharepoint-webviewer-sample.js

```diff
@@ -62,13 +62,19 @@ const installPdfTronWebViewer = () => createProcessPromise(NPM_COMMAND, [
   '--prefix',
   'pdftron-webpart-sample',
   'i',
-  '@pdftron/webviewer',
+  '@pdftron/webviewer@7.3.3',
```

Comment:
I guess we're not sure if this will work on the latest WV 8?

Reply:
Yeah, I just haven't had time to test `webviewer@8.0.0` on Sharepoint thoroughly, so I figure we should play it safe